content/projects/data-science-specific/oop-for-data-science/_index.md

```diff
@@ -15,6 +15,9 @@ story_points: 5
 submission_type: repo
 tags:
 - oop-data-sci
+- plotly
+- seaborn
+- sklearn
```

Comment:
```suggestion
- linear-regression
```
Not sure if we need a specific `oop-data-sci` tag here or if a general `oop` tag will suffice. @EckardBerry @Andy-Nkumane thoughts?
FYI: @sheenarbw